data/en/spreadsheetaddinfo.json

```diff
@@ -18,6 +18,15 @@
 	},
 	"links": [
 
+	],
+	"examples": [
+		{
+			"title":"Tag Example",
+			"description":" ",
+			"code":"<cfspreadsheet action=\"read\" src=\"#filename#\" name=\"a\" > \r\n <cfset info = StructNew()> \r\n <cfset info.title=\"Title\"> \r\n <cfset info.category=\"Category test\"> \r\n <cfset info.author=\"ABC\"> \r\n <cfset info.comments=\"Comments for this file\"> \r\n <cfset spreadsheetaddInfo(a,info)> \r\n <cfspreadsheet action=\"write\" filename=\"#dirname#SingleSheet.xls\" name=a overwrite=\"yes\">  ",
```

Comment:
Same as adobe doc, ideally the example here could omit the first and last lines where cfspreadsheet is used, and just show the struct.